cinema/models.py

```diff
@@ -121,3 +112,16 @@ def __str__(self):
 
     class Meta:
         unique_together = ("movie_session", "row", "seat")
```

Comment:
To maintain better organization within your code, it's advisable to position the class Meta either before or after all the method definition